src/archanaNJun20/CodingExam10.java

```diff
@@ -0,0 +1,18 @@
+package archanaNJun20;
+
+public class CodingExam10 {
+	void printRotation(String input1, String input2) {
+		int count = 0;
+		while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && count< input1.length)